feature/home/src/main/kotlin/uk/govuk/app/home/ui/HomeScreen.kt

```diff
@@ -134,7 +134,7 @@ private fun ScalingHeader(
     Column(modifier = modifier) {
         Image(
             painter = painterResource(id = uk.govuk.app.design.R.drawable.logo),
-            contentDescription = null,
+            contentDescription = "GOV.UK Logo",
```

Comment:
Wondering if this might need going in a string resource file maybe ready for Welsh translation of logo? (not sure what logo is in Welsh?)

Reply:
Happy to add this as a string resource. However, according to Google translate at least - the Welsh for `Logo` is `Logo`, though it is pronounced differently to the English version.

As an aside - when read by TalkBack - it appends the word `image` to that phrase. So, we get `GOV.UK Logo image`. This seems build-in and so not changeable. One for Post Amigo I think.